packages/react-native/src/primitives/Radio/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+
+import { RadioStyles } from './types';
+
+export const styles: RadioStyles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+  },
+  radio: {
+    height: 30,
+    width: 30,
+    borderRadius: 30,
+    borderWidth: 2,
+    padding: 5,
```

Comment:
We want to avoid using odd numbers for assigning space related values, they prevent centering of layout items when used in theme structures. Our default spacing size will be `8`, and the rest of the values will be derived based on that. Suggest using `4` here